syncopy/tests/helpers.py

```diff
@@ -19,30 +19,30 @@
 def run_padding_test(method_call, pad_length):
     """
     The callable should test a solution and support
-    a single keyword argument `pad_to_length`
+    a single keyword argument `pad`
     """
 
-    pad_options = [pad_length, 'nextpow2', None]
+    pad_options = [pad_length, 'nextpow2', 'maxperlen']
     for pad in pad_options:
-        method_call(pad_to_length=pad)
+        method_call(pad=pad)
 
     # test invalid pads
     try:
-        method_call(pad_to_length=2)
+        method_call(pad=0.1) # trials should be longer than 0.1 seconds
```

Comment:
I suggest changing this to `pad = -0.1` or something else. Trials of 0.1 seconds length aren't completely outrageous.

Reply:
Hehe, that's in the eye of the beholder.. but jokes aside, agreed !